### DISCUSSION TITLE
Check dashboard resources

### DISCUSSION THREAD

**Discussion by frenck** [frenck/spook#534] [Ideas]
Dashboard resources could have errors:

- Link to missing resources
- Loading non-existing resources
- Duplicate resources

Would be nice if Spook raises an issue for those.

**Comment by Sennevds** [comment]
Maybe also resources that aren't used. I install several cards to try out but sometime forget to remove them. After a while and many dashboards it's a lot of manual work to find out which resources I'm still using
